python_modules/libraries/dagster-components/dagster_components/core/dsl_schema.py

```diff
@@ -30,16 +27,7 @@ class AssetAttributesModel(BaseModel):
     code_version: Optional[str] = None
     owners: Sequence[str] = []
     tags: Union[str, Mapping[str, str]] = {}
-    automation_condition: Optional[Union[str, AutomationCondition]] = RenderingScope(
-        Field(None), required_scope={"automation_condition"}
-    )
-
-    class Config:
-        # required for AutomationCondition
-        arbitrary_types_allowed = True
-
-    def get_resolved_attributes(self, value_resolver: TemplatedValueResolver) -> Mapping[str, Any]:
-        return value_resolver.render_obj(self.model_dump(exclude_unset=True))
+    automation_condition: Optional[str] = None
```

Comment:
For my own edification, where is the code that makes this "work" now?  How do I understand the behavior of the rendering?

Reply:
Tl;dr I was expecting _some_ sort of metadata or self-description here that makes it clear what I can do in these templates.

Reply:
updated